airflow/task/standard_task_runner.py

```diff
@@ -183,8 +177,7 @@ def _start_by_fork(self):
                 return_code = 1
 
                 self.log.exception(
-                    "Failed to execute job %s for task %s (%s; %r)",
-                    job_id,
+                    "Failed to execute task %s (%s; %r)",
```

Comment:
```suggestion
                    "Failed to execute task id=%s exception=%s pid=%r",
```
May as well improve to this or similar.  Cus key-values are better in tools like splunk.  As opposed to just the pid being an int and we have no info about what it is.

Reply:
Also on this one: since this is `log.exception` we don't need to include exc again